.travis.yml

```diff
@@ -85,9 +85,8 @@ after_success:
 deploy:
   # Deploy Docs to Github Pages when tagged as release
   provider: pages
-  strategy: git
-  cleanup: false
-  token: $GITHUB_TOKEN
+  skip_cleanup: true
```

Comment:
`skip_cleanup` is deprecated, that's why I used `cleanup` lest time

Reply:
Yeah, it's strange. TravisCI gives a warning when you use this setting, but using just `cleanup` doesn't work. See the last release log: https://travis-ci.org/github/rakutentech/ios-miniapp/builds/715699238

The docs for the pages deployment still have the old values, so I think they haven't updated this plugin yet: https://docs.travis-ci.com/user/deployment/pages/#further-configuration